### ISSUE TITLE
win32 installer creates an useless coil.bat file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3150]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3150|
|Type|enhancement|
|Created|2008-04-02 20:02:40Z|

That should be fixed.

Attachments:

* [3150.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a12/a1254acc3431c64117c1412d3713945c2e668dc2/863e006113f7be18028ce09541be6151c23bd62b.diff) (632 bytes) - added by therve on 2008-04-02 20:03:43Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3150 3150
type__enhancement enhancement
reporter__therve therve
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1207166560000000 1207166560000000
changetime__1207514470000000 1207514470000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to @therve|
|-|-|

I guess this is fine. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [23219]) Fix various issues with win32 build scripts:
 * Re-include zope.interface
 * Remove useless coil.bat script
 * Add an icon to the installer

Author: therve
Reviewer: radix
Fixes [#3144](https://github.com/twisted/twisted/issues/3144)
Fixes [#3150](https://github.com/twisted/twisted/issues/3150)
Fixes [#3151](https://github.com/twisted/twisted/issues/3151)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
